test/support/acceptance_case.ex

```diff
@@ -5,18 +5,23 @@ defmodule Ex338.AcceptanceCase do
     quote do
       use Wallaby.DSL
 
-      import Ecto.Schema
-      import Ecto.Query, only: [from: 2]
-      import Ex338.Router.Helpers
+      alias Ex338.Repo
+      import Ecto
+      import Ecto.Changeset
+      import Ecto.Query
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives